cryptonite.cabal

```diff
@@ -103,6 +103,11 @@ Flag check_alignment
   Default:           False
   Manual:            True
 
+Flag use_target_attributes
+  Description:       use GCC / clang function attributes instead of global target options.
+  Default:           False
```

Comment:
I'm tempted to have this default to `true` here and let people with old compiler has the pain & flag to fallback to the old behavior ?

Reply:
Thanks for the push, I was tempted too.